src/main/pg/gps_rescue.c

```diff
@@ -29,7 +29,7 @@
 
 #include "gps_rescue.h"
 
-PG_REGISTER_WITH_RESET_TEMPLATE(gpsRescueConfig_t, gpsRescueConfig, PG_GPS_RESCUE, 6);
+PG_REGISTER_WITH_RESET_TEMPLATE(gpsRescueConfig_t, gpsRescueConfig, PG_GPS_RESCUE, 7);
```

Comment:
Ah... I had forgotten that now we are using position_control altitude PIDs for this, so that I should have removed the throttle PIDs here - I thought I'd already done that.  fixed, and PG should be 7 now due to the change.